core/modules/field_ui/field_ui.module

```diff
@@ -140,25 +140,50 @@ function field_ui_menu() {
             'weight' => 2,
             'file' => 'field_ui.admin.inc',
           ) + $access;
-
+          // Secondary tab for 'Manage display' overview.
+          $items["$path/display/overview"] = array(
+            'title' => 'Overview',
+            'type' => MENU_DEFAULT_LOCAL_TASK,
+            'weight' => -99,
+          ) + $access;
           // Display modes.
           // The same base $path for the menu item (with a placeholder) can be
           // used for all bundles of a given entity type; but depending on
           // administrator settings, each bundle has a different set of view
           // modes available for customization. So we define menu items for all
           // Display modes, and use an access callback to determine which ones
           // are actually visible for a given bundle.
-          $items["$path/display/%"] = array(
-            'page callback' => 'backdrop_get_form',
-            'page arguments' => array('field_ui_display_form', $entity_type, $bundle_arg, $view_mode_position),
-            // The access callback needs to check both the current 'custom
-            // display' setting for the Display mode, and the overall access
-            // rules for the bundle admin pages.
-            'access callback' => '_field_ui_view_mode_menu_access',
-            'access arguments' => array_merge(array($entity_type, $bundle_arg, $view_mode_position, $access['access callback']), $access['access arguments']),
-            'type' => MENU_LOCAL_TASK,
-            'file' => 'field_ui.admin.inc',
-          );
+          $view_modes = array(
+            'default' => array(
+              'label' => t('Default'),
+            ),
+          ) + $entity_info['view modes'];
+
+          foreach ($view_modes as $view_mode => $view_mode_info) {
+            $items["$path/display/$view_mode"] = array(
+              'title' => $view_mode_info['label'],
+              'page callback' => 'backdrop_get_form',
+              'page arguments' => array(
+                'field_ui_display_form',
+                $entity_type,
+                $bundle_arg,
+                $view_mode,
+              ),
+              // The access callback needs to check both the current 'custom
+              // display' setting for the view mode, and the overall access
+              // rules for the bundle admin pages.
+              'access callback' => '_field_ui_view_mode_menu_access',
+              'access arguments' => array_merge(
+                array(
+                  $entity_type,
+                  $bundle_arg,
+                  $view_mode,
+                  $access['access callback'],
+                ), $access['access arguments']),
```

Comment:
The formatting here was done in order to make PHPCS happy, but it made me pause for a moment. Since the order arrays are added in `array_merge()` does not matter, can we please do it this way?:
```suggestion
              'access arguments' => array_merge(
                $access['access arguments'],
                array(
                  $entity_type,
                  $bundle_arg,
                  $view_mode,
                  $access['access callback'],
                ),
              ),
```

Reply:
@klonos, the order of the array under the key `'access arguments'` INDEED matters. The change you are suggesting will break the callback `_field_ui_view_mode_menu_access()`.

That callback expects a minimum of 4 arguments, in the correct order: `$entity_type, $bundle, $view_mode, $access_callback`. Beyond that, it can take additional arguments, retrieved via `$args = array_slice(func_get_args(), 4);` in `_field_ui_view_mode_menu_access()` , and sent to the new `access_callback` function located in the 4th place of the arguments for `_field_ui_view_mode_menu_access()`.

So, no, we can't make the change you are suggesting, sorry.

Reply:
Unless I'm missing something, I will resolve this comment without making this change. Please feel free to reopen if indeed I'm missing something here.